### PR TITLE
Devops/separate test types 132

### PR DESCRIPTION
**Pull request by szirbucz**
Move integration tests to separate folder and maven profile

### CODE REVIEW

**Comment by magwas** [comment]
I see this in the build output: [WARNING] No input file found for /root/src/github.com/edemo/PDEngine/src/integration-test/java/org/rulez/demokracia/pdengine/integrationtest/VoteCreationIntegrationTest.java
And I cannot see the integration tests running.

File: Makefile [hunk 1]
```diff
@@ -42,7 +42,7 @@ javabuild: target/PDEngine-0.0.1-SNAPSHOT.jar
 target/PDEngine-0.0.1-SNAPSHOT.jar:
 	mvn build-helper:parse-version versions:set versions:commit -DnewVersion=\$${parsedVersion.majorVersion}.\$${parsedVersion.minorVersion}.\$${parsedVersion.incrementalVersion}-$$(tools/getbranch|sed 'sA/A_Ag').$$(git rev-parse --short HEAD)
 	mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent install org.pitest:pitest-maven:mutationCoverage site
-	zenta-xslt-runner -xsl:cpd2pmd.xslt -s:target/pmd.xml -o target/pmd_full.xml
+	zenta-xslt-runner -xsl:cpd2pmd.xslt -s:target/pmd.xml -o target/pmd_full.xml -Pintegration-test
```
Comment (magwas) [inline Makefile]: didn't you mean the previous line?

**Comment by magwas** [comment]
Now I see the integration tests. But still got this:
[WARNING] No input file found for /root/src/github.com/edemo/PDEngine/src/integration-test/java/org/rulez/demokracia/pdengine/integrationtest/VoteCreationIntegrationTest.java. No PMD issue will be imported on this file.

**Comment by magwas** [comment]
I still see the above

File: tools/pullanalize [hunk 2]
```diff
@@ -22,6 +22,7 @@ then
         -Dsonar.github.login=magwas\
         -Dsonar.externalIssuesReportPaths=issuesReport.json\
         -Dsonar.java.pmd.reportPaths=target/pmd_full.xml\
+        -Dsonar.tests=src/test,src/integration-test\
```
Comment (magwas) [inline tools/pullanalize:25]: Isn't it needed in the other sonar run as well?

**Review by magwas — CHANGES_REQUESTED** [review]

File: tools/pullanalize (hunk 2 above)
Comment (szirbucz) [reply tools/pullanalize:25]: It is, but at first I wanted to try it. I couldn't reproduce it with the same mvn command on my machine.

**Comment by magwas** [comment]
Do you need any help?

**Review by magwas — APPROVED** [review]

**Comment by magwas** [comment]
testcasecount=5

**Pull request merged**